modules/i18n/src/main/I18nKeys.scala

```diff
@@ -1501,6 +1501,25 @@ object I18nKeys:
     val `sendEmailAt` = I18nKey("contact:sendEmailAt")
     val `explainYourRequest` = I18nKey("contact:explainYourRequest")
 
+  object punishments:
```

Comment:
These are not punishments, we don't do punitions.

They're account constraints due to ToS violations.

A better name for the object could be `tos` or `appeal`.